### PR TITLE
LPS-79496 don't suppress these properties lets find a better way to match expressions

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

**Comment by gamerson** [comment]
ci:test

**Review by Ithildir — CHANGES_REQUESTED** [review]

File: modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplateFilesTest.java [hunk 1]
```diff
@@ -161,8 +173,9 @@ private void _testArchetypeMetadataXml(
 			Path projectTemplateDirPath, String projectTemplateDirName,
 			Path archetypeResourcesDirPath, Properties bndProperties,
 			boolean requireAuthorProperty, String archetypePostGenerateGroovy,
-			Set<String> archetypeResourcePropertyNames)
-		throws IOException {
+			Set<String> archetypeResourceExpressions,
+			DocumentBuilder documentBuilder)
+		throws IOException, SAXException {
```
[the remark this answers is not in the document]
Comment (gamerson) [reply modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplateFilesTest.java]: ok.

**Comment by gamerson** [comment]
ci:retest

**Comment by gamerson** [comment]
ci:retest

**Comment by Ithildir** [comment]
ci:stop Sent to https://github.com/brianchandotcom/liferay-portal/pull/58840.

**Pull request closed without merging**